src/main/java/gdx/liftoff/ui/RootTable.java

```diff
@@ -108,7 +91,7 @@ public void transitionTable(int tableIndex, boolean rightToLeftTransition) {
         //animation initial setup
         clearChildren();
         stage.addActor(table);
-        add(newTable).prefSize(rootWidth, rootHeight);
+        add(newTable).prefSize(600, 700);
```

Comment:
Should these also be ROOT_TABLE_PREF_WIDTH and ROOT_TABLE_PREF_HEIGHT, instead of 600 and 700? I can change them after merging.